image/v1/types.go

```diff
@@ -61,6 +61,19 @@ type Image struct {
 	DockerImageManifestMediaType string `json:"dockerImageManifestMediaType,omitempty" protobuf:"bytes,9,opt,name=dockerImageManifestMediaType"`
 	// DockerImageConfig is a JSON blob that the runtime uses to set up the container. This is a part of manifest schema v2.
 	DockerImageConfig string `json:"dockerImageConfig,omitempty" protobuf:"bytes,10,opt,name=dockerImageConfig"`
+	// DockerImageManifests holds information about sub-manifests when the image represents a manifest list.
+	DockerImageManifests []ImageManifest `json:"dockerImageManifests,omitempty" protobuf:"bytes,11,rep,name=dockerImageManifests"`
```

Comment:
Are this field and DockerImageLayers mutually exclusive?

Reply:
Yes

Reply:
You should make sure to add comments explaining that within your API. Do you publish openapi schemas for this API?

Reply:
Not sure about schemas. @dmage do you happen to know?

Reply:
These are native objects, they use Swagger Doc. I doubt we have an explicit OpenAPI schema for them.

Reply:
Related https://github.com/openshift/api/pull/1252/files

Reply:
Any comments you add here will also end up in the swagger docs too. If this is end user facing, you should add comments to explain to the user that they shouldn't be setting both at the same time

Reply:
tbh, user shouldn't create these objects. They should be created via creating virtual `ImageStreamImport` or `ImageStreamMapping` objects.